pkg/apiserver/handlers_test.go

```diff
@@ -200,6 +200,35 @@ func TestTimeout(t *testing.T) {
 	}
 }
 
+func TestHasApiPrefix(t *testing.T) {
```

Comment:
Ok. there is no test for `GetAttribs()` yet, any hints on how to mock `requestAttributeGetter`. Are there factories for this or should I implement interfaces. A pointer to where to peek in the source code would be fine.

Also, is it considered good practice to put helper fn's on structs to group stuff? `hasApiPrefix()` does not modify the state of the `requestAttributeGetter` obj, so it could just be a func.